File: code-coaches/medium/thats-odd/ThatsOdd.cs
using System;

class ThatsOdd
{
     static int SumEvenNumbers(int[] numbers)
     {
        int sum = 0;

        foreach (int num in numbers)
        {
            if (num % 2 == 0)
                sum += num;
        }

        return sum;
    }

    static void Main (string[] args)
    {
        int numbersNum = int.Parse(Console.ReadLine());
        int[] numbers = new int[numbersNum];

        for (int i = 0; i < numbersNum; i++)
        {
            numbers[i] = int.Parse(Console.ReadLine());
        }

        Console.WriteLine(SumEvenNumbers(numbers));
    }
}

File: code-coaches/medium/thats-odd/ThatsOdd.swift
func sumEvenNumbers(numbers: [Int]) -> Int {
    var sum = 0
    for num in numbers {
        if num % 2 == 0 {
            sum += num
        }
    }
    return sum
}

func main() {
    let numbers_num = Int(readLine()!)!
    var numbers: [Int] = []
    for _ in 0..<numbers_num {
        numbers.append(Int(readLine()!)!)
    }
    print(sumEvenNumbers(numbers: numbers))
}

main()
